tests/hstestcase.py

```diff
@@ -71,6 +71,33 @@ def _unset_testbotgroup(cls):
         cls.project.settings.apidelete('botgroups')
         cls.project.settings.expire()
 
+    def start_job(self, **startparams):
+        jobdata = self.project.jobq.start(**startparams)
+        if jobdata:
+            jobkey = jobdata.pop('key')
+            jobauth = (jobkey, jobdata['auth'])
+            return self.project.get_job(jobkey, jobauth=jobauth, metadata=jobdata)
+
+    def job_finished(self, job, close_reason=None):
```

Comment:
can be replaced in most places directly by call to `self.hsclient.jobq.finish(job)`